i18n/en.json

```diff
@@ -322,29 +321,39 @@
   "exhibit updated": "Exhibit updated",
   "exhibit saved": "Exhibit saved",
 
-  "exhibit page": "Page",
+  "mobile pages": "Mobile Pages",
   "Appetizer_Page": "Appetizer Page",
   "Image_Page": "Image Page",
   "Slider_Page": "Slider Page",
   "Text_Page": "Text Page",
-  "delete page": "Delete page",
-  "confirm delete page": "Do you really want to delete page '{{title}}'?",
-  "create exhibit page": "Create mobile page",
+  "additional information pages": "Additional information pages",
   "hide year numbers": "Hide year numbers",
   "images": "Images",
+  "font": "Font",
+  "DEFAULT": "Default",
+  "AlteSchwabacher": "Alte Schwabacher",
+  "add page": "Add page",
+  "page added": "Page added",
+  "addition failed": "Could not add",
+  "create page": "Create mobile page",
   "page created": "Page created",
   "create failed": "Could not create",
-  "page deleted": "Page deleted",
-  "delete failed": "Could not delete",
+  "delete page": "Delete page",
+  "confirm delete page": "Do you really want to remove page '{{title}}' from the entire system?",
+  "page deleted": "Page removed",
+  "delete failed": "Could not remove",
+  "unset page": "Remove page",
+  "confirm unset page": "Do you really want to remove page '{{title}}' from this exhibit?",
+  "edit page": "Edit page",
   "page updated": "Page updated",
   "update failed": "Could not update",
   "page load failed": "Could not retrieve pages",
   "no pages": "No pages found",
-  "font": "Font",
-  "DEFAULT": "Default",
-  "AlteSchwabacher": "Alte Schwabacher",
   "no title": "no title",
   "page order updated": "Page order updated",
+  "select page": "Select page",
+  "no info pages": "no additional pages",
```

Comment:
Because this line actually appears in the editor in parentheses as "(no additional pages)" and after a colon. Rules of punctuation tell me to not start with a capital in this case. And it looks better.